### DISCUSSION TITLE
Can't see a pushed package

### DISCUSSION THREAD

**Discussion by marek-lime** [community/community#51586] [Packages]
### Select Topic Area

Question

### Body

In our company, we are maintaining the [garlic](https://github.com/Lundalogik/garlic) project. Recently, we've tried pushing a new version of the nuget package created from this project to a github package repo. We're pushing the package with the dotnet nuget push command, specifying the source and access token as the key parameter. The commands runs successfully, but for some reason we cannot see the package, neither on the UI, nor in our IDEs. When we try to push it again with the same version, we get 409, which would mean that a package was already pushed. Please advise!

**Comment by baynezy** [comment]
Did you ever resolve this? I have the same problem.

**Reply by marek-lime** [reply]
Yes, it turned out to be an access rights issue. Only the owners of the repository within the organization where the package was pushed had permission to view it. After a call with one of the owners, we confirmed that the package had indeed been pushed, so the 409 status code was accurate. To simplify access for others, we subsequently obtained read permissions.

**Reply by baynezy** [reply]
Ah OK that is interesting. I am pushing packages to a user account not an organisation so that is very weird.

Thanks for responding.
